src/start.clj

```diff
@@ -27,34 +41,28 @@
                     (.start connection)
                     (.send producer message)
                     (.commit session))))))]
-    (with-push-to-cloud-jms-connection url queue username password send)))
-
-(defn receive-text
-  "Nil on timeout or the text from a message from the push-to-cloud JMS queue with URL, QUEUE, USERNAME and PASSWORD"
-  [url queue username password]
-  (letfn [(receive [connection destination]
-            (let [transacted? false
-                  timeout 10000]
-              (with-open [session (.createSession connection transacted? Session/AUTO_ACKNOWLEDGE)]
-                (let [queue (.createQueue session destination)]
-                  (with-open [consumer (.createConsumer session queue)]
-                    (.start connection)
-                    (.receive consumer timeout))))))]
-    (with-push-to-cloud-jms-connection url queue username password receive)))
+    (send connection destination)))
 
-(defn plus
-  "Return the sum of A and B. Just for testing."
-  [^Integer a ^Integer b]
-  (+ a b))
```

Comment:
Not sure what this is trying to log, but maybe it should include COUNTER.
Maybe just EDNify the TextMessage and then log it?  Maybe as JSON?
Why sleep? Use a durable consumer, block in receive, don't time out.
Then we don't need CADENCE either.

Reply:
Thanks, this is the exact insight I wanted! Maybe we could talk more about the EDNify stuff offline, but I do agree that we probably want to block in receive, which will make the parallelization later easier.

Reply:
OK I ended up using `prn-str`